app/javascript/articles/Feed.jsx

```diff
@@ -221,9 +222,9 @@ export const Feed = ({ timeFrame, renderFeed, afterRender }) => {
         },
         credentials: 'same-origin',
       }),
-      fetch(`/display_ads/feed_first`),
-      fetch(`/display_ads/feed_second`),
-      fetch(`/display_ads/feed_third`),
+      fetch(`/display_ads/feed_first?user_tags=${userTags}`),
+      fetch(`/display_ads/feed_second?user_tags=${userTags}`),
+      fetch(`/display_ads/feed_third?user_tags=${userTags}`),
```

Comment:
I am curious why we're appending the tags in the URL, rather than query them controller-side?

Reply:
The alternative is to always send the user tags as an argument for  the for_display method in the display_ads_controller when we encounter any of the feed placement areas.

That felt like I was hardcoding it a bit, hence I chose to send it through with the query which i felt made it more dynamic, but it could work to query it on the backend too if you feel that's preferential?

Reply:
I've had trouble following the product spec here — is the idea that _only_ home feed ads can target `user-following-tags` ? (Even if that's the idea, I'm not totally sure the best place to encode that restriction is in the JS URL composition...)